process/factory/shard/vmContainerFactory.go

```diff
@@ -15,12 +15,12 @@ import (
 	"github.com/ElrondNetwork/elrond-go/process/factory/containers"
 	"github.com/ElrondNetwork/elrond-go/process/smartContract/hooks"
 	vmcommon "github.com/ElrondNetwork/elrond-vm-common"
-	arwen12 "github.com/ElrondNetwork/wasm-vm-v1_2/arwen"
-	arwenHost12 "github.com/ElrondNetwork/wasm-vm-v1_2/arwen/host"
-	arwen13 "github.com/ElrondNetwork/wasm-vm-v1_3/arwen"
-	arwenHost13 "github.com/ElrondNetwork/wasm-vm-v1_3/arwen/host"
-	arwen14 "github.com/ElrondNetwork/wasm-vm-v1_4/arwen"
-	arwenHost14 "github.com/ElrondNetwork/wasm-vm-v1_4/arwen/host"
+	wasm12 "github.com/ElrondNetwork/wasm-vm-v1_2/arwen"
```

Comment:
Consider replacing `wasm1x` with `wasmvm1x` because `wasm1x` sounds like it's a VM that executes a specific version of the WASM spec (i.e. the version of the WASM language).

Reply:
+1

Reply:
done